2006-12-01  Paul Eggert  <eggert@cs.ucla.edu>

	* gzip.c (abort_gzip_signal): Don't call abort () as a "safeguard",
	since it actually causes a core dump.  Problem reported by
	Jakub Bogusz.

--- gzip-1.3.6/gzip.c.orig	2006-12-03 18:41:59.066521760 +0100
+++ gzip-1.3.6/gzip.c	2006-12-03 18:42:05.874909748 +0100
@@ -1854,5 +1854,4 @@
    remove_output_file ();
    signal (sig, SIG_DFL);
    raise (sig);
-   abort ();
 }
